fdbserver/DataDistribution.actor.cpp

```diff
@@ -480,7 +480,10 @@ struct DataDistributor : NonCopyable, ReferenceCounted<DataDistributor> {
 
 			wait(yield(TaskPriority::DataDistribution));
 		}
+		return Void();
+	}
 
+	ACTOR static Future<Void> resumeFromInitDataMoveMap(Reference<DataDistributor> self) {
```

Comment:
Same here - can the name simply be initDataMoveMap?

Reply:
I want to retain the `resumeFrom` which describes the intention more accurately in my opinion.